scrapers/TeamskeetAPI.py

```diff
@@ -39,63 +33,77 @@ def save_json(api_json, url):
         pass
 
 
+USERFOLDER_PATH = str(pathlib.Path(__file__).parent.parent.absolute())
+DIR_JSON = os.path.join(USERFOLDER_PATH, "scraperJSON","Teamskeet")
+
+
+# Not necessary but why not ?
+USER_AGENT = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0'
+
+
 fragment = json.loads(sys.stdin.read())
 if fragment["url"]:
     scene_url = fragment["url"]
 else:
-    debug('You need to set the URL (e.g. teamskeet.com/movies/*****)')
+    log.error('You need to set the URL (e.g. teamskeet.com/movies/*****)')
     sys.exit(1)
 
 if "teamskeet.com/movies/" not in scene_url:
-    debug('The URL is not from a Teamskeet URL (e.g. teamskeet.com/movies/*****)')
+    log.error('The URL is not from a Teamskeet URL (e.g. teamskeet.com/movies/*****)')
     sys.exit(1)
 
 scene_id = re.sub('.+/', '', scene_url)
 if not scene_id:
-    debug("Error with the ID ({})\nAre you sure that the end of your URL is correct ?".format(scene_id))
+    log.error("Error with the ID ({})\nAre you sure that the end of your URL is correct ?".format(scene_id))
     sys.exit(1)
 use_local = 0
 json_file = os.path.join(DIR_JSON, scene_id+".json")
 if os.path.isfile(json_file):
-    print("Using local JSON...", file=sys.stderr)
+    log.debug("Using local JSON...")
     use_local = 1
     with open(json_file, encoding="utf-8") as json_file:
         scene_api_json = json.load(json_file)
 else:
-    print("Asking the API...", file=sys.stderr)
+    log.debug("Asking the API...")
     api_url = 'https://store2.psmcdn.net/ts-elastic-d5cat0jl5o-videoscontent/_doc/{}'.format(
         scene_id)
     headers = {
         'User-Agent': USER_AGENT,
         'Origin': 'https://www.teamskeet.com',
         'Referer': 'https://www.teamskeet.com/'
     }
-
+    scraper = cloudscraper.create_scraper()
     # Send to the API
     r = ""
     try:
-        r = requests.get(api_url, headers=headers, timeout=(3, 5))
+        r = scraper.get(api_url, headers=headers, timeout=(3, 5))
     except requests.exceptions.RequestException:
```

Comment:
```suggestion
    except cloudscraper.exceptions.CloudflareException:
```
It's not request for sure, but not sure if this one is the correct exception.

Reply:
yea sure